plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/inspector-controls/linked-product-control.tsx

```diff
@@ -129,10 +133,17 @@ const getFromCurrentProductRadioLabel = (
 	currentLocation: string,
 	isUsesReferenceIncludesCart: boolean
 ): string => {
-	if ( currentLocation === 'cart' && isUsesReferenceIncludesCart ) {
+	if (
+		currentLocation === REFERENCE_TYPE_CART &&
+		isUsesReferenceIncludesCart
+	) {
 		return __( 'From products in the cart', 'woocommerce' );
 	}
 
+	if ( currentLocation === REFERENCE_TYPE_ORDER ) {
```

Comment:
Is there a reason why a similar `isUsesReferenceIncludesOrder` isn't needed here? Alternatively, is the cart-related one necessary in the first place?

Reply:
Wow! Great catch. We need it for Order context. I will add it. Thanks for pointing it out 🙇‍♂️